modules/gapi/src/api/render.cpp

```diff
@@ -2,7 +2,6 @@
 
 #include <stdexcept>
 
-#include <opencv2/imgproc.hpp>
```

Comment:
Could you add this header to render.hpp ?

Reply:
I can, but `render.cpp` include `render.hpp` from `gapi/opencv_gapi/include/opencv2/gapi/`. 

This `render.hpp` is just a shortcut to `render/render.hpp`.

`#include <opencv2/imgproc.hpp>` already exist in `render/render.hpp`.